File: Linkpoint/src/main/java/com/linkpoint/slproto/modules/rlv/commands/RLVCmdRezObjects.java
package com.linkpoint.slproto.modules.rlv.commands;

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType;

public class RLVCmdRezObjects extends RLVCmdGenericRestriction {
    public RLVCmdRezObjects() {
        super(RLVRestrictionType.rez, false);
    }
}

File: Linkpoint/src/main/kotlin/com/linkpoint/slproto/modules/rlv/commands/RLVCmdRezObjects.kt
package com.linkpoint.slproto.modules.rlv.commands

import com.linkpoint.slproto.modules.rlv.RLVRestrictionType

class RLVCmdRezObjects : RLVCmdGenericRestriction() {
    public RLVCmdRezObjects() {
        super(RLVRestrictionType.rez, false)
    }
}
